esignet-core/src/main/java/io/mosip/esignet/core/dto/ClientDetailCreateRequestV3.java

```diff
@@ -10,6 +10,8 @@
 @Data
 @NoArgsConstructor
 public class ClientDetailCreateRequestV3 extends ClientDetailCreateRequestV2 {
+
+    @ClientAdditionalConfigConstraint
```

Comment:
We have not named any validator as Constraint, following the same naming convention I think we should keep the name as "ClientAdditonalConfig"